packages/aws-cdk-lib/aws-fsx/README.md

```diff
@@ -52,6 +51,33 @@ const fileSystem = new fsx.LustreFileSystem(this, 'FsxLustreFileSystem', {
 });
 ```
 
+### File System Type Version
+
+You can set [the Lustre version for the file system](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/aws-resource-fsx-filesystem.html#cfn-fsx-filesystem-filesystemtypeversion). To do this, use the `fileSystemTypeVersion` property:
+
+```ts
+declare const vpc: ec2.Vpc;
+
+const fileSystem = new fsx.LustreFileSystem(this, 'FsxLustreFileSystem', {
+lustreConfiguration: { deploymentType: fsx.LustreDeploymentType.SCRATCH_2 },
+  storageCapacityGiB: 1200,
+  vpc,
+  vpcSubnet: vpc.privateSubnets[0],
+  fileSystemTypeVersion: fsx.FileSystemTypeVersion.LUSTRE_2_15,
+});
+```
+
+**Note**: The `fileSystemTypeVersion` has a restrictions on the values that can be set based on the `deploymentType`.
+
+- `V_2_10` is supported by the Scratch and `PERSISTENT_1` deployment types.
+- `V_2_12` is supported by all Lustre deployment types, except for `PERSISTENT_2` with a metadata configuration mode.
+- `V_2_15` is supported by all Lustre deployment types and is recommended for all new file systems.
+
+**Note**: The default value of `fileSystemTypeVersion` is `V_2_10` except for the following cases:
+
+- Default value is `V_2_12` when `deploymentType` is set to `PERSISTENT_2` without a metadata configuration mode.
+- Default value is `V_2_15` when `deploymentType` is set to `PERSISTENT_2` with a metadata configuration mode.
```

Comment:
I have included descriptions related to metadata configuration mode, but the current `LustreFileSystem` construct does not allow setting metadata. Should descriptions related to metadata be included?

Reply:
I thought it might be good to either add a note stating that Metadata Configuration cannot be set yet, or alternatively, include the metadata configuration property in this PR. 
What do you think?

The [MetadataConfiguration](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-fsx-filesystem-lustreconfiguration.html#cfn-fsx-filesystem-lustreconfiguration-metadataconfiguration) property already exists, so it seems possible to add it. However, since it's a new property, it appears that descriptions haven't been defined for it yet.

Reply:
Thank you for your opinion!

I think it is better to support `MetadataConfiguration` initially, I was planning to address the `metadataConfiguration` in this PR as well, but since all the CloudFormation documentation was marked as "not available," I decided to try again once that issue is resolved.

I have updated the README to reflect the assumption that `metadataConfiguration` is not configured.